### PR TITLE
Adding the fix for the webflow 

### LINKED ISSUE #756: Fatal error: Uncaught TypeError: is_link(): Argument #1 ($filename) must be of type string, Drush\Config\Environment given in /tmp/site/vendor/webflo/drupal-finder/src/DrupalFinder.php:153
# Bug report
Phpstan is failing due to code updated in the webflo : https://github.com/webflo/drupal-finder/pull/65/files
Phpstan is using the this module
error :
Fatal error: Uncaught TypeError: is_link(): Argument #1 ($filename) must be of type string, Drush\Config\Environment given in /tmp/site/vendor/webflo/drupal-finder/src/DrupalFinder.php:153

### PR DESCRIPTION
**Pull request by DEVDESAGAR1**
Adding the fix for the webflow 

fixes #756

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "phpstan/phpstan-deprecation-rules": "^1.1.4",
         "symfony/finder": "^4.2 || ^5.0 || ^6.0 || ^7.0",
         "symfony/yaml": "^4.2|| ^5.0 || ^6.0 || ^7.0",
-        "webflo/drupal-finder": "^1.2"
+        "webflo/drupal-finder": "^1.3"
```
Comment (mglaman) [inline composer.json:18]: I'd like to avoid bumping the minimum and support both 1.2 and 1.3

**Comment by mglaman** [comment]
Well, that sucks. \DrupalFinder\DrupalFinderComposerRuntime::getVendorDir is picking up InstalledVersions from within the PHPStan phar so it's broken with getVendorDir

**Comment by webflo** [comment]
> Well, that sucks. \DrupalFinder\DrupalFinderComposerRuntime::getVendorDir is picking up InstalledVersions from within the PHPStan phar so it's broken with getVendorDir

Lets try to harden this part. Another approach I have tried some time ago was an composer plugin which dumps the config.

https://github.com/webflo/drupal-finder-plugin/blob/1.x/src/DrupalFinderPlugin/Installer.php#L163

**Comment by mglaman** [comment]
@webflo thanks for looking into this, I paused and did a pivot at DrupalCon

**Comment by weitzman** [comment]
Any chance we can fix this soon? Would help at https://github.com/ddev/ddev-drupal-contrib/pull/53

**Comment by webflo** [comment]
@weitzman Could you try https://github.com/webflo/drupal-finder/pull/73 thanks! This is my preferred approach. Thanks!

**Comment by weitzman** [comment]
Unfortunately I dont have time to test phpstan with the PR. And Drush no longer uses drupal-finder.

**Comment by webflo** [comment]
I have created a new release for drupal-finder

**Comment by mglaman** [comment]
Thanks @webflo!

**Comment by webflo** [comment]
Superseded by https://github.com/mglaman/phpstan-drupal/pull/790

**Pull request closed without merging**